src/Player/Camera/Camera.gd

```diff
@@ -1,7 +1,86 @@
-extends Camera
+extends Spatial
 
 var _target: Spatial
+export var camera_rotation_speed: = Vector2(2.5, 2.5)
+export var yinvert_multiplier: = -1
```

Comment:
Is this necessary? Isn't a multiplier that inverts a value always -1?

Reply:
Different people have different inversion preferences. Exposing a value that developers can set in the editor to control default behaviour feels like the right way to go.

Making it use an enum seems to provide a dropdown in the editor, which is a bit clearer. I'll switch it up to use that.